src/dope-node/communication/communication_handler.go

```diff
@@ -0,0 +1,126 @@
+package communication
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/gorilla/websocket"
+)
+
+var knownAddresses = []string{}
+
+func RegisterEndpoints(boostrapServerAddress string, isBoostrapServer bool, port int) {
+	http.HandleFunc("/ws", wsHandler)
+	log.Println("Running server")
+	if isBoostrapServer {
+		http.HandleFunc("/addresses", httpHandler)
+	} else {
+		// This sends 'hello' message to all known nodes
+		addresses := fetchNodeAddresses(boostrapServerAddress, "127.0.0.1:"+strconv.Itoa(port))
```

Comment:
magic string, why do we have a `127.0.0.1` here

Reply:
Its temporary for now (and probably until moment of real-deployment)